consensus/istanbul/backend/backend.go

```diff
@@ -549,7 +551,11 @@ func (sb *Backend) Commit(proposal istanbul.Proposal, aggregatedSeal types.Istan
 			return err
 		}
 	}
-	go sb.onNewConsensusBlock(block, result.Receipts, result.Logs, result.State)
+	sb.wg.Add(1)
```

Comment:
Doesn't this waitgroup face the same problem with the concurrent `Add` ? or are Commit and Close both called from the same thread?